src/Symfony/Component/Ldap/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+* Added a new option called `extra_fields` as an array it will allow you to ask for more information that will be saved in the metadata field of the user, so you can get field value as an array and anything that is in the ldap.
```

Comment:
What about something shorter? `Added the "extra_fields" option, an array of custom fields to pull from the LDAP server`